### PR TITLE
fix: always show current version in chart history

### PR DESCRIPTION
**Pull request by ZeRego**
Closes: #7306

### Description:

Before:

Screenshot 2023-10-12 at 12 54 00

With fix:

Screenshot 2023-10-12 at 12 50 46

( still filters older versions )
Screenshot 2023-10-12 at 12 50 01

### Reviewer actions

- [ ] I have manually tested the changes in the preview environment
- [ ] I have reviewed the code
- [ ] I understand that "request changes" will block this PR from merging

### CODE REVIEW

File: packages/backend/src/models/SavedChartModel.ts [hunk 1]
```diff
@@ -359,13 +372,21 @@ export class SavedChartModel {
...
-                `${SavedChartVersionsTableName}.created_at`,
-                '>=',
-                moment().subtract(60, 'days'),
-            )
+            .andWhere(function () {
+                // get all versions from the last X days + the current version ( in case is older than X days )
+                this.where(
+                    `${SavedChartVersionsTableName}.created_at`,
+                    '>=',
+                    moment().subtract(60, 'days'),
```
Comment (IrakliJani) [inline packages/backend/src/models/SavedChartModel.ts:384]: we can use `current_timestamp - interval '60 days'` here

**Review by IrakliJani — APPROVED** [review]
looks good, left one suggestion - not a blocker

File: packages/backend/src/models/SavedChartModel.ts (hunk 1 above)
Comment (ZeRego) [reply packages/backend/src/models/SavedChartModel.ts:384]: Going to try that in https://github.com/lightdash/lightdash/pull/7450

Comment (IrakliJani) [reply packages/backend/src/models/SavedChartModel.ts:384]: 👌

**Pull request merged**